src/Compilers/CSharp/csc/Program.cs

```diff
@@ -8,6 +9,14 @@ namespace Microsoft.CodeAnalysis.CSharp.CommandLine
     public class Program
     {
         public static int Main(string[] args)
-            => BuildClient.RunWithConsoleOutput(args, RequestLanguage.CSharpCompile, Csc.Run);
+        {
```

Comment:
It was kind of charming that we had an EXE which had an expression bodied member "Main" method.
Why can't clientDir and workingDir be supplied in the BuildClient helper method?